crypto/asn1/a_bitstr.c

```diff
@@ -142,6 +142,10 @@ ASN1_BIT_STRING *ossl_c2i_ASN1_BIT_STRING(ASN1_BIT_STRING **a,
  */
 int ASN1_BIT_STRING_set_bit(ASN1_BIT_STRING *a, int n, int value)
 {
+    if (n < 0) {
+        return 0;
+    }
```

Comment:
Just:
```c
if (n < 0)
    return 0;
```
our code style omits the braces for a single statement body.

Reply:
I've applied the necessary changes. Thanks for the info!